files/zh-tw/webassembly/reference/control_flow/block/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: block
+slug: WebAssembly/Reference/Control_flow/block
+tags:
+  - WebAssembly
+  - wasm
+  - Landing page
+  - Reference
+  - Control flow
```

Comment:
Hi @shiyou0130011, could you remove this part? We do not need those metadata in translated-content.

For now, please see: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE.

Reply:
Hello @yin1999 
I've removed them